wandb/sdk/wandb_init.py

```diff
@@ -1152,6 +1152,30 @@ def _monkeypatch_tensorboard() -> None:
     tb_module.patch()
 
 
+def try_create_root_dir(settings: Settings):
```

Comment:
always annotate the return type, even if it's None

```suggestion
def try_create_root_dir(settings: Settings) -> None:
```

otherwise, mypy may not typecheck the function